README.md

```diff
@@ -67,3 +87,132 @@ npm run grade -- --challenge stockPriceChart
 - 3: Clean design
 - 4: Well-designed with good UX
 - 5: Exceptional design
+
+## Collaborative Grading System
+
+### Current State vs. Ideal Structure
+
+**Current Issues:**
+- Grader saves results to `grader-app/public/benchmarks/results/results.csv` (isolated)
+- No git integration for collaborative contributions
+- Manual file copying between main repo and grader app
+- Results aren't versioned or shared between graders
+
+**Ideal Collaborative Structure:**
+
+```
+benchmarks/
+├── results/
+│   ├── results.csv              # Main results file (git-tracked)
+│   ├── grades/                  # Individual grader contributions
+│   │   ├── alice-2024-01-15.csv # Timestamped grader files
+│   │   ├── bob-2024-01-16.csv
```

Comment:
This is a great idea. I'd love to see grade files stack up in the repo over time. Then we can load them all in at once for analysis/visualization combined.